src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -925,6 +925,8 @@ def remove_properties(instance, argument_values):
         except IndexError:
             raise CLIError('index {} doesn\'t exist on {}'
                            .format(list_index, list_attribute_path[-1]))
+        except AttributeError:
+            raise CLIError('{} doesn\'t exist'.format(list_attribute_path[-1]))
```

Comment:
On a second thought, we can use `if not list_to_remove_from: raise ...`. But it's OK. No difference.